tests/testsuite.src/used_binaries.at

```diff
@@ -271,7 +271,7 @@ AT_CHECK([$GREP 'PARAGRAPH_EX_l_7:'    prog.c], [0], ignore, [])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [bluBb END], [])
 
 AT_CAPTURE_FILE([prog.d])
```

Comment:
As the purpose of this test is (primarily) "check that cobc implies the right options depending on the output file specified" (and the output file _after your change **corectly**_ isn't even generated because of `-M`) and we now have a dependency generation test: please move this series of checks down to the new test.